### PR TITLE
fixing softmax dim for attention pooling

### PR DESCRIPTION
**Pull request by NicoBach**
Hello,
with the most recent version of pytorch the attention pooling of the LAMA-actor in https://github.com/Scitator/neurips-18-prosthetics-challenge (catalyst@ aefc813) is broken. The softmax function is computed on the wrong dimension.

In the following picture the attention pooling yields 1 for each of the 4 observations due to the above: 
Screenshot(softmax=dim1)

Computing softmax with dim=2 fixes this issue:
Screenshot_softmax(dim=2)

Co-authored-by: Andrew Melnik <andrew.melnik.git@gmail.com>

### CODE REVIEW

**Comment by Scitator** [comment]
Hi,

NeurIPS-18 repo is currently in archive mode and would work only with the proposed catalyst version (catalyst @ https://github.com/catalyst-team/catalyst/commit/aefc813f16198f82cc748afc6d21b3122aab3671) and PyTorch 0.4.1.
All other version are not supported. Nevertheless, you can wait for this year solution with updated catalyst and pytorch versions.
